src/main/java/uk/gov/digital/ho/hocs/queue/common/BaseMessageHandler.java

```diff
@@ -1,19 +1,26 @@
 package uk.gov.digital.ho.hocs.queue.common;
 
+import lombok.extern.slf4j.Slf4j;
+
 import java.util.List;
 
+@Slf4j
 public abstract class BaseMessageHandler implements MessageHandler {
 
-    private final List<String> ignoredMessageTypes;
+    private final boolean shouldIgnoreMessage;
 
     protected BaseMessageHandler(List<String> ignoredMessageTypes) {
-        this.ignoredMessageTypes = ignoredMessageTypes;
+        shouldIgnoreMessage = ignoredMessageTypes.stream()
+                .map(String::toUpperCase)
+                .anyMatch(messageType -> messageType.equals(getMessageType().getType().toUpperCase()));
```

Comment:
There is an equals ignore case

Reply:
or just ignoreMessageTypes.contains() since we control the message and the list.